sdk/purview/Azure.Analytics.Purview.Scanning/src/Generated/PurviewDataSourceClient.cs

```diff
@@ -72,6 +72,52 @@ public PurviewDataSourceClient(Uri endpoint, string dataSourceName, TokenCredent
         /// <summary> Creates or Updates a data source. </summary>
         /// <param name="content"> The content to send as the body of the request. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <example>
+        /// This sample shows how to call CreateOrUpdateAsync with required request content and parse the result.
+        /// <code><![CDATA[
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("<https://my-account-name.azure.com>");
+        /// var client = new PurviewDataSourceClient(endpoint, credential);
```

Comment:
This is also a subclient.

Reply:
fixed by using `CodeGenClient` attribute, the side effect is that `PurviewDataSourceClient` to `PurviewDataSource`.